newIDE/app/src/SceneEditor/ToolbarCommands.js

```diff
@@ -16,6 +16,8 @@ type Props = {|
   toggleWindowMask: () => void,
   toggleGrid: () => void,
   setupGrid: () => void,
+  canSceneObjectRename: boolean,
+  onSceneObjectRename: () => void,
```

Comment:
Reusing the naming of the `Toolbar` props, it should be `canRenameObject` and `onRenameObject`.
Moreover, it should not contain a notion of scene here as you should be able to rename both a scene object and a global object

Reply:
Done!